### PR TITLE
fix dock missing install qmlplugin

### PR DESCRIPTION
**Pull request by 18202781743**
- fix: missing files for dock plugin
- chore: update changelog to 0.0.3

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,11 @@
+dde-shell (0.0.3) unstable; urgency=medium
```
Comment (wineee) [inline debian/changelog:1]: unreleased ?

**Review by wineee — CHANGES_REQUESTED** [review]

**Review by wineee — APPROVED** [review]

**Review by asterwyx — APPROVED** [review]

**Pull request merged**